lib/menu.rb

```diff
@@ -0,0 +1,52 @@
+class Menu
+  attr_accessor :todays_menu
+
+  def initialize(menu_items)
+    @menu_items = menu_items
+    @todays_menu = [] # Instead of array, maybe use hash??
+
+    # This conditional is probably not needed, but it
+    # doesn't do harm (actually, it might) and it serves as a
+    # reminder to me that this kind of checking is possible
+    # and is sometimes desirable. I got my inspiration from:
+    # https://www.ruby-lang.org/en/documentation/quickstart/3/
+    if @menu_items.nil?
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.